Microsoft.Alm.Authentication/Src/Git/Utilities.cs

```diff
@@ -131,9 +136,12 @@ internal static IEnumerable<Win32.ProcessEntry32> EnumerateParentProcesses()
             {
                 processEntry = processEntries[processId];
 
-                processList.AddFirst(processEntry);
+                processList.Add(processEntry);
```

Comment:
So the list is now reversed? You were adding to the front of a `LinkedList` but are now added to the end of a `List`.
Does the caller expect a particular order?